src/main/java/mesosphere/marathon/client/Marathon.java

```diff
@@ -24,10 +26,16 @@
 	GetAppsResponse getApps();
 
 	@RequestLine("GET /v2/apps?cmd={cmd}&id={id}&label={labelSelector}")
-	GetAppsResponse filterApps(@Param("cmd") String command,
-			@Param(value = "id", expander = AppIdNormalizer.class) String id,
+	GetAppsResponse filterApps(@Param("cmd") String commandSubstring,
+			@Param(value = "id", expander = AppIdNormalizer.class) String idSubstring,
 			@Param("labelSelector") String labelSelector);
 
+	@RequestLine("GET /v2/apps?cmd={cmd}&id={id}&label={labelSelector}&embed={embed}")
+	GetAppsResponse filterApps(@Param("cmd") String commandSubstring,
+			@Param(value = "id", expander = AppIdNormalizer.class) String idSubstring,
+			@Param("labelSelector") String labelSelector,
+			@Param("embed") EnumSet<EmbeddedAppResource> embeddedResources);
```

Comment:
Instead of demanding the concrete `EnumSet<EmbeddedAppResource>` type here, instead accommodate the wider type `Set<EmbeddedAppResource>`. We could widen it all the way out to `Iterable<EmbeddedAppResource>`, but that then loses the suggestion that there's no sense in repeating any of the potential values.